src/gui/qgsgui.h

```diff
@@ -87,12 +89,18 @@ class GUI_EXPORT QgsGui
      */
     static QgsLayoutItemGuiRegistry *layoutItemGuiRegistry();
 
+    /**
+     *
+     */
+    static void enableAutoPostionRestore( QWidget *widget, const QString &key = QString() );
```

Comment:
Postion->Position

Actually "enabledAutoGeometryRestore" is probably a better name, since it's also the size which is restored.

Reply:
Yep good point. Fixing now.